rclrs/src/node/publisher.rs

```diff
@@ -107,6 +108,30 @@ where
         })
     }
 
+    /// Returns the topic that the publisher is publishing data to
+    pub fn get_topic(&self) -> Result<String, RclrsError> {
+
+        let mut _topic_name: String = String::default();
+        // SAFETY: No preconditions for the functions called.
+        // Edge cases for all unsafe variables created have been handled
+        // The unsafe variables created get dropped as soon as they're used
+        unsafe {
+            let raw_topic_pointer = rcl_publisher_get_topic_name((&mut *self.handle.lock()) as *mut _);
```

Comment:
The `as *mut _` can be omitted, see https://github.com/ros2-rust/ros2_rust/pull/134.

Reply:
ohh, okay, the declaration had this type so I just assumed it was required... I've made the necessary changes